lib/travis/hub/api/sentry.rb

```diff
@@ -0,0 +1,12 @@
+require 'raven'
+
+module Travis
+  module Hub
+    class Sentry < Sinatra::Base
+      configure do
+        Raven.configure { |c| c.tags = { environment: environment } }
```

Comment:
And perhaps also the equivalent of https://github.com/travis-ci/travis-api/blob/484e5baee3fbc872e6461842b6100e9d57b1a6f1/lib/travis/api/app/middleware/request_id.rb, that can wait until after this PR though. :)